egs/wsj/s5/steps/nnet/train_mmi.sh

```diff
@@ -68,6 +68,10 @@ if ! $skip_cuda_check; then cuda-compiled || { echo "Error, CUDA not compiled-in
 
 mkdir -p $dir/log
 
+utils/lang/check_phones_compatible.sh $lang/phones.txt $srcdir/phones.txt
+utils/lang/check_phones_compatible.sh $lang/phones.txt $alidir/phones.txt
```

Comment:
this line is repeated.
Also (this comment reflects some other scripts in steps/nnet/), if a script has set -e in it, you don't need to do "|| exit 1", and it's prettier if you remove it.